pkg/apis/provisioning/v0alpha1/types.go

```diff
@@ -32,6 +32,14 @@ type S3RepositoryConfig struct {
 	// TODO: How do we define access? Secrets?
 }
 
+type Workflow string
+
+const (
+	PullRequestWorkflow Workflow = "pull-request"
+	BranchWorkflow      Workflow = "branch"
+	PushWorkflow        Workflow = "push"
```

Comment:
```suggestion
    // Create a pull request when saving changes
	PullRequestWorkflow Workflow = "pull-request"
	// Create a new branch with changes
	BranchWorkflow      Workflow = "branch"
	// Push directly to the configured branch
	PushWorkflow        Workflow = "push"
```

did i get the descriptions right?